### PR TITLE
You can now crack your knuckles. crack!

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

Adds the ability to crack your knuckles with *crack

Credit to MrHugo13
https://github.com/tgstation/tgstation/pull/50060

#### Changelog

:cl:  
rscadd: Adds knuckle cracking
/:cl:

### CODE REVIEW

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
1. Link the PR you ported in the description, that being: https://github.com/tgstation/tgstation/pull/50060

2. Rohesie didn't make it, MrHugo13 did.

**Comment by swissloaf** [comment]
> 1. Link the PR you ported in the description, that being: [tgstation/tgstation#50060](https://github.com/tgstation/tgstation/pull/50060)
> 2. Rohesie didn't make it, MrHugo13 did.

Done and done

File: code/modules/mob/living/carbon/emote.dm [hunk 1]
```diff
@@ -34,6 +34,18 @@
 							'sound/misc/clap3.ogg',
 							'sound/misc/clap4.ogg')
 
+/datum/emote/living/carbon/crack
+	key = "crack"
+	key_third_person = "cracks"
+	message = "cracks their knuckles."
+	sound = 'sound/misc/knuckles.ogg'
+	cooldown = 6 SECONDS
```
Comment (Hopekz) [inline code/modules/mob/living/carbon/emote.dm:42]: Nice someone's using my cooldown framework PR

**Review by Hopekz — APPROVED** [review]

File: code/modules/mob/living/carbon/emote.dm [hunk 2]
```diff
@@ -34,6 +34,18 @@
...
+
+/datum/emote/living/carbon/crack/can_run_emote(mob/living/carbon/user, status_check = TRUE , intentional)
+	if(user.get_num_arms() <= 1)
```
Comment (alexkar598) [inline code/modules/mob/living/carbon/emote.dm]: This code is ugly
```suggestion
	if(user.get_num_arms() < 2)
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/emote.dm (hunk 2 above)
Comment (swissloaf) [reply code/modules/mob/living/carbon/emote.dm]: you can literally pop with one hand

Comment (alexkar598) [reply code/modules/mob/living/carbon/emote.dm]: The message is cracks their knuckles with an s, stop being a cunt

**Review by alexkar598 — CHANGES_REQUESTED** [review]
<2 is easier to read than <= 1

**Pull request merged**